lib/internal/auth.ts

```diff
@@ -27,3 +27,16 @@ export const getAccessToken = async (
 
   return Promise.resolve(response);
 };
+
+export const getWebComponentToken = async (
```

Comment:
We do not want to couple this to checkout. You can use it for other resources and multiple resources (https://docs.justifi.tech/infrastructure/webComponentTokens)